.github/mergeable.yml

```diff
@@ -50,3 +50,8 @@ mergeable:
         changed:
           file: 'package.json'
           files: ['yarn.lock']
+      # If config.yml is updated, so should default_airflow.cfg
+      - do: dependent
```

Comment:
I was too late (just started to write this comment) I think this might be not good check. If someone modifies default_airflow.cfg and changes value or add space, probot will complain. I guess pre-commit-config should be more than enough.

Reply:
Yup, I am going to follow up with a couple more PRs on it for polishing. I will remove it from mergeable in that one

Reply:
But at least now that we have got this merged, need configs can directly go to yml with description :)

Reply:
Withdraw. I see that all of it is generated automatically, so no risk here.